ndbench-core/src/main/java/com/netflix/ndbench/core/generators/DefaultDataGenerator.java

```diff
@@ -103,7 +103,7 @@ private void initialize(int lowerBound, int upperBound) {
     private void initialize() {
         for (int i = 0; i < config.getNumValues(); i++) {
             if(i%1000==0) {
```

Comment:
For future, you may want to do (i%1000 && config.getNumValues() > 1000) || ( i%100 && config.getNumValues()< 100))` to cover cases that somebody has configured very low number of values.